source/agora/consensus/Validation.d

```diff
@@ -1002,7 +1002,7 @@ public string isInvalidReason (const ref Block block, in ulong prev_height,
         - The block height must be at least unlock_height or greater of the UTXO.
 
     Params:
-        block_height = the height of the direct ancestor of this block
+        block_height = Enter the height of the block to verified
```

Comment:
`the height at which the enrollment wants to be added to the block`

Reply:
applied the suggestion.